### PR TITLE
Version 0.1

### PR DESCRIPTION
**Pull request by Peter1907**
Hello, dear code reviewer!

In this project I've:

- Built the required app according to the project requirements
- Styled the app according to the design guideline provided
- Updated the README file to include all necessary tools for setting up the project & the main features
- Also to include an acknowledgment for the design's author
- Made an introductory video explaining everything

Here is the video
https://www.loom.com/share/3c2aecc55fb04252a1cc9d29e5a9a9ca

I mentioned in the video I couldn't deploy the project to Heroku nor to Render as neither provides a free option
but I've found a way to deploy to Render

Let me know if there's anything to be added or modified regarding the code.
Thank you so much for your time & effort.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+- Athentication (using devise)
+- Image upload, validation & processing (using "shrine" & "image_processing" gems)
+- Responsive design with tailwind
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- Coming soon when the project is finished
+
```
Comment (Mihndim2020) [inline README.md:55]: - [ ] As is a [requirement ](https://github.com/microverseinc/curriculum-rails/blob/main/capstone/rails_capstone.md)for this project, "`Technical requirements: The project should be deployed and accessible online.`" Please deploy your application using either `Heroku, Netlify or GitHub Pages`.
Please make sure your project is deployed properly, the home and detail page should work as expected. A link to the deployed project is present in the README. I understand from your video presentation that you have some challenges to deploy your app due to payments, the above mentioned platform have free tiers, please kindly use one of them. Checkout this [link ](https://devcenter.heroku.com/articles/getting-started-with-rails7)on how to deploy your app with Heroku.

**Review by Mihndim2020 — CHANGES_REQUESTED** [review]
### Hi @Peter1907, 

### Good job so far!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Good commit messages
:heavy_check_mark: Descriptive Readme file
:heavy_check_mark:  Good job with video presentation
:heavy_check_mark: Good job with test coverage
:heavy_check_mark:  Clean and organized code
:heavy_check_mark: Good job with UI

There are some issues that you still need to work on to get this project approved, but you are almost there!
ALMOST-MINION

# Status: Requires Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

### N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear or you can get to me on slack  using this handle @mihndim.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (Peter1907) [reply README.md:55]: @Mihndim2020  Unfortunately, Heroku doesn't allow me to sign up even & after asking some of my friends, they told me that Heroku is no longer free.
Netlify & GitHub pages can't deploy rails applications.
So what now? :joy:

File: README.md [hunk 2]
```diff
@@ -51,7 +51,7 @@ The application is a fully functional app that allows users to sign-up/log-in, c
 
 ## 🚀 Live Demo <a name="live-demo"></a>
 
-- Coming soon when the project is finished
+- [Live Demo](https://blue-ray-pay.onrender.com)
```
Comment (rloterh) [inline README.md:54]: - [ ] Nice job for deploying your app! However, I'm unable to successfully signup/login to the deployed app. I get the below error showing in the browser console whiles attempting to signup. Kindly ensure to improve upon this aspect. You can also consider other [free] hosting sites for hostinfg fullstack apps such as _'railway.app'_

**Signup/Login error:**
image

File: README.md (hunk 1 above)
Comment (rloterh) [reply README.md:55]: Needs attention!

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Peter, 

You did extremely great on this project! 👏 👏 
There are however a few issues that you still need to work on to go to the next project but you are almost there! 🚦 🚦  🏃 

## Highlights 👍  
- Good job introducing improvements!

## Required Changes ♻️
_Kindly check the inline comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

**You can also consider:**
- You used incorrect git-flow/ GitHub flow for this project. Kindly note that a capstone project worked on by a single contributor should have merge request flow **from feature into development**. Kindly note this important step for subsequent projects.

Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 2 above)
Comment (rloterh) [reply README.md:54]: resolved! ✔️

**Review by rloterh — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi Peter,  

Your project is complete! 🏁 Great job! There is nothing else to say other than... it's time to merge it 🍾 Congratulations!  🎉💯🌟 

## Highlights 👍
- Nice job implementing recommended changes!

Keep soaring higher! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**